afs/afs-ws/afs-ws-server-utils/src/main/java/com/powsybl/afs/ws/server/utils/AppDataBean.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.ws.server.utils;
+
+import com.powsybl.afs.AfsException;
+import com.powsybl.afs.AppData;
+import com.powsybl.afs.AppFileSystem;
+import com.powsybl.afs.ProjectFile;
+import com.powsybl.afs.storage.ListenableAppStorage;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.computation.DefaultComputationManagerConfig;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@Named
+@Singleton
+public class AppDataBean {
+
+    protected AppData appData;
+
+    protected ComputationManager shortTimeExecutionComputationManager;
+
+    protected ComputationManager longTimeExecutionComputationManager;
+
+    public AppData getAppData() {
+        return appData;
+    }
+
+    public ListenableAppStorage getStorage(String fileSystemName) {
+        ListenableAppStorage storage = appData.getRemotelyAccessibleStorage(fileSystemName);
+        if (storage == null) {
+            throw new WebApplicationException("App file system '" + fileSystemName + "' not found",
+                    Response.Status.NOT_FOUND);
+        }
+        return storage;
+    }
+
+    public AppFileSystem getFileSystem(String name) {
+        Objects.requireNonNull(appData);
+        AppFileSystem fileSystem = appData.getFileSystem(name);
+        if (fileSystem == null) {
+            throw new WebApplicationException("App file system '" + name + "' not found", Response.Status.NOT_FOUND);
+        }
+        return fileSystem;
+    }
+
+    public <T extends ProjectFile> T getProjectFile(String fileSystemName, String nodeId, Class<T> clazz) {
+        Objects.requireNonNull(fileSystemName);
```

Comment:
If you consider the previous remark, this check is not needed anymore

Reply:
Removed